schema-next.yaml

```diff
@@ -2,6 +2,13 @@ file_format: 1.1.0
 schema_url: https://opentelemetry.io/schemas/next
 versions:
   next:
+    logs:
+      changes:
+        # https://github.com/open-telemetry/semantic-conventions/pull/908
+        - rename_attributes:
+            attribute_map:
+              log.file.name: file.name
```

Comment:
FYI @djaglowski we would need to inherit this breaking change in the filelog receiver as well. Most probably through a feature flag? Would that be doable?

Reply:
I think that one is resolved now after the conclusion at https://github.com/open-telemetry/semantic-conventions/pull/908#issuecomment-2059918201.
Thank's @djaglowski for the input here!